app/filtering.js

```diff
@@ -305,7 +305,7 @@ function registerPermissionHandler (session, partition) {
         }
       }
     }
-    const url = origin || embedderOrigin
+    const url = mainFrameUrl || origin
```

Comment:
so I think this is incorrect too @diracdeltas because there will always be an origin and that is how we want to lookup the permission